draft-ietf-httpbis-digest-headers.md

```diff
@@ -1461,6 +1440,27 @@ Location: /authors/123
 {"id": "123", "author": "Camilleri"}
 ~~~
 
+# Changes from RFC3230
+
+## Deprecate Negotiation of Content-MD5 {#deprecate-contentMD5}
+
+This RFC deprecates the negotiation of Content-MD5 as it has been obsoleted by
+[RFC7231].
+The `contentMD5` token defined in Section 5 of [RFC3230] MUST NOT be used as a digest-algorithm.
```

Comment:
We can't have normative text in the appendix. IIUC mnot suggested putting this section in the IANA considerations.

Reply:
Split in two sections:
- `contentMD5` in IANA
- negotiation in Changes from 3230